contracts/ListaStakeManager.sol

```diff
@@ -284,15 +286,19 @@ contract ListaStakeManager is
      * @param _idx - The index of the request in the array returns by getUserWithdrawalRequests()
      */
     function claimWithdraw(uint256 _idx) external override whenNotPaused {
-        claimWithdrawFor(msg.sender, _idx);
+        _claimWithdrawFor(msg.sender, _idx);
     }
 
     /**
-     * @dev This function allows to claim the requested withdrawals for other users
+     * @dev This function allows to claim the requested withdrawals for other users; only bot can call this function
      * @param _user - The address of the user who raised WithdrawRequest
      * @param _idx - The index of the request in the array returns by getUserWithdrawalRequests()
      */
-    function claimWithdrawFor(address _user, uint256 _idx) public override whenNotPaused {
+    function claimWithdrawFor(address _user, uint256 _idx) external override whenNotPaused onlyRole(BOT) {
+        _claimWithdrawFor(_user, _idx);
+    }
+
+    function _claimWithdrawFor(address _user, uint256 _idx) internal {
```

Comment:
it's better to use `private`

Reply:
done in 8611264f92f5ded9cab5cfed00807467297e8869